### DISCUSSION TITLE
swapping all #includes for local filepath

### DISCUSSION THREAD

**Discussion by anomaliesintent112** [nlohmann/json#3747] [Q&A]
I don't know why I was forced to do this but trying to compile a c++ program with 
x86_64-w64-mingw32-g++ -o wave64.exe wavetry2.cc
forced me to change all instances of #include <{filepath}> to "/usr/local/include/nlohmann/{rest of path}"
it took me 30 minutes because I had to do this for all files and all instances but once i was done it worked without a problem and it didnt break g++ either anyways if anyone ever needs the files let me know and ill fork it and upload my versions

**Comment by falbrechtskirchinger** [comment]
https://gcc.gnu.org/onlinedocs/gcc/Directory-Options.html

```
-I/usr/local/include
```

No need to replace paths.
